src/common/DataUtils.cpp

```diff
@@ -685,6 +754,10 @@ DataSpace hostAccessibleDataSpace(DataSpace dataSpace)
     case DataSpace::CudaDevice:
       return DataSpace::CudaPinned;
 
+    case DataSpace::SyclManaged:
+    case DataSpace::SyclDevice:
+      return DataSpace::SyclPinned;
+
```

Comment:
add DataSpace::SyclPinned above

Reply:
I believe it's handled properly now if you want to take another look.